### PR TITLE
DNS suffix option mention

### PR DESCRIPTION
**Pull request by cap1984**

### CODE REVIEW

File: deploy/cr.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@ metadata:
   name: my-cluster-name
 spec:
 #  platform: openshift
+#  ClusterServiceDNSSuffix: svc.cluster.local
```
Comment (delgod) [inline deploy/cr.yaml]: the following should work
```suggestion
#  clusterServiceDNSSuffix: svc.cluster.local
```

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| self-healing | passed |
| init-deploy | passed |
| storage | passed |
| demand-backup | failed |
| monitoring | failed |
| limits | passed |
| scaling | failed |
| operator-self-healing | passed |
| one-pod | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| self-healing | passed |
| init-deploy | passed |
| storage | passed |
| demand-backup | failed |
| monitoring | passed |
| limits | passed |
| scaling | failed |
| monitoring-2-0 | passed |
| operator-self-healing | passed |
| arbiter | passed |
| one-pod | failed |
| service-per-pod | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| self-healing | passed |
| monitoring | passed |
| operator-self-healing | passed |
| monitoring-2-0 | passed |
| one-pod | failed |
| arbiter | passed |
| service-per-pod | passed |
| init-deploy | passed |
| limits | passed |
| scaling | passed |
| demand-backup | failed |

**Pull request merged**